introduction.md

```diff
@@ -1,3 +1,9 @@
 My prefered path on codeschool is Ruby.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?